canonical-websites/maas.io
# Update the contact form content and layout

## Done

Updated the contact us form

## QA
 - Go to [http://0.0.0.0:8006/contact-us](http://0.0.0.0:8006/contact-us)
 - See that the content matches the [copydoc](https://docs.google.com/document/d/12zxzauq9GV6NYVTeKCCqtwMpflRGZewE5fdTg1SZoOQ/edit#)

Fixes #274
